src/Common/Infrastructure/RegistryApiController.cs

```diff
@@ -18,6 +20,9 @@ public abstract class RegistryApiController<T> : ApiController<T>
 
         protected readonly IFeatureToggle CacheToggle;
 
+        protected abstract string GoneExceptionMessage { get; }
+        protected abstract string NotFoundExceptionMessage { get; }
```

Comment:
To be honest, I'm not sure if this is a good way to deal with the message. This limits the message to 1 message per controller, if we ever need a different message we're stuck and have to override HandleBadRequest.

Wouldn't it be nicer to have a factory style method return a `HandleBadRequest` customized with the strings? (the ones you are now setting in the ctor)

Reply:
Maybe something like:

```
    protected Action<HttpStatusCode> BuildBadRequestHandler(string notAcceptableMessage = "Ongeldig formaat.", string badRequestMessage = "", string goneMessage = "", string notFoundMessage = "")
        {
return ....
        }
```

And a caller would do:
```
BuildBadRequestHandler(goneMessage: "specific thingie")
```